pkg/gds/emails/templates/reissuance_reminder.txt

```diff
@@ -0,0 +1,22 @@
+Hello {{ .Name }},
+
+We wish to remind you that your TRISA Identity Certificates for {{ .CommonName }} will soon be expiring. The expiration date on your current certificates is {{ .ExpirationDate }}.
+
+The Global Directory Service will automatically reissue your certificates on {{ .ReissueDate }}. When the certificates are reissued, you will receive two additional emails; one of which will contain your password-protected certificates and the other of which will contain your PKCS12 password, which you will use to decrypt the new certificates.
```

Comment:
I would re-work the second and third sentences this way (taking out the semicolon): 
"When the certificates are reissued, you will receive two additional emails - one will contain your password-protected certificates and the other will contain your PKCS12 password. You will use the PKCS12 password to decrypt the new certificates."